service/src/main/resources/quips/insultibot/insults.json

```diff
@@ -537,5 +504,214 @@
     "tags":[],
     "targetableText":"I talked to Santa Claus, and he says it's not within his power to bring [TARGET] a real friend for Christmas.",
     "targetableSsml":"I talked to Santa Claus, and he says it's not within his power to bring [TARGET] a real friend for Christmas."
+  },
+  {
+    "quipGroup":"QUITE_SIMPLE",
+    "text":"If someone were to ask me to describe you, it would be, quite simple.",
+    "ssml":"If someone were to ask me to describe you, it would be, quite simple.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"If someone were to ask me to describe [TARGET], it would be, quite simple.",
+    "targetableSsml":"If someone were to ask me to describe [TARGET], it would be, quite simple."
+  },
+  {
+    "quipGroup":"ALPHABET_SOUP",
+    "text":"A bowl of alphabet soup has a larger vocabulary than you.",
+    "ssml":"A bowl of alphabet soup <break time=\"50ms\" /> has a larger vocabulary than you.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"A bowl of alphabet soup has a larger vocabulary than [TARGET].",
+    "targetableSsml":"A bowl of alphabet soup <break time=\"50ms\" /> has a larger vocabulary than [TARGET]."
+  },
+  {
+    "quipGroup":"BACON_CANNIBALISM",
+    "text":"Bacon tastes great. It's a shame a pig like you can't have any without it being cannibalism.",
+    "ssml":"Bacon tastes great. <break /> It's a shame a pig like you <break time=\"50ms\" /> can't have any without it being cannibalism.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"Bacon tastes great. It's a shame a pig like [TARGET] can't have any without it being cannibalism.",
+    "targetableSsml":"Bacon tastes great. <break /> It's a shame a pig like [TARGET] <break time=\"50ms\" /> can't have any without it being cannibalism."
+  },
+  {
+    "quipGroup":"NO_FAITH",
+    "text":"I once had faith in humanity. Then I met you.",
+    "ssml":"I once had faith in humanity. <break /> Then I met you.",
+    "usageRules":[
+      
+    ],
+    "tags":[],
+    "targetableText":"I once had faith in humanity. Then I met [TARGET].",
+    "targetableSsml":"I once had faith in humanity. <break /> Then I met [TARGET]."
+  },
+  {
+    "quipGroup":"HURRICANE",
+    "text":"They should name a hurricane after you, because you're a disaster.",
```

Comment:
We should probably just have a quipGroup called "disasters".  I bet there are various other disaster metaphors we could come up with.